apps/sms/views/conversation/js/conversation.js

```diff
@@ -513,16 +513,16 @@ var ConversationView = {
     // get an event whenever the panel changes?
     Threads.currentId = args.id;
 
-    var prevPanel = args.meta.prev && args.meta.prev.panel;
+    var prevPanel = args.meta.prev;
 
     // If transitioning from composer, we don't need to notify about type
     // conversion but only after the type of the thread is set
     // (afterEnterThread)
-    if (prevPanel !== 'composer') {
+    if (!prevPanel || prevPanel.panel !== 'composer') {
       this.enableConvertNoticeBanners();
     }
 
-    if (prevPanel !== 'group-view' && prevPanel !== 'report-view') {
+    if (!prevPanel || !this.isConversationPanel(args.id, prevPanel)) {
```

Comment:
It seems this ```if``` can include the rest of the code below, but decided not to change much here.